ansible/roles/schulcloud-server-core/templates/common-cartridge-deployment.yml.j2

```
@@ -0,0 +1,97 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: common-cartridge-deployment
+  namespace: {{ NAMESPACE }}
+  labels:
+    app: common-cartridge
+    app.kubernetes.io/part-of: schulcloud-verbund
+    app.kubernetes.io/version: {{ SCHULCLOUD_SERVER_IMAGE_TAG }}
+    app.kubernetes.io/name: common-cartridge
+    app.kubernetes.io/component: files
+    app.kubernetes.io/managed-by: ansible
+    git.branch: {{ SCHULCLOUD_SERVER_BRANCH_NAME }}
+    git.repo: {{ SCHULCLOUD_SERVER_REPO_NAME }}
+spec:
+  replicas: {{ COMMON_CARTRIDGE_REPLICAS|default("1", true) }}
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 1
+      #maxUnavailable: 1
+  revisionHistoryLimit: 4
+  paused: false
+  selector:
+    matchLabels:
+      app: common-cartridge
+  template:
+    metadata:
+      labels:
+        app: common-cartridge
+        app.kubernetes.io/part-of: schulcloud-verbund
+        app.kubernetes.io/version: {{ SCHULCLOUD_SERVER_IMAGE_TAG }}
+        app.kubernetes.io/name: common-cartridge
+        app.kubernetes.io/component: files
```

This will be part from the Server so the component shoud be server and not files
```suggestion
        app.kubernetes.io/component: server
```

---

done ✅